ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-igmp-snooping.j2

```diff
@@ -1,14 +1,12 @@
 {# eos - IP IGMP Snooping #}
-{% if ip_igmp_snooping is arista.avd.defined %}
-{%     if ip_igmp_snooping.globally_enabled is arista.avd.defined(false) %}
+{% if ip_igmp_snooping.globally_enabled is arista.avd.defined(false) %}
 !
 no ip igmp snooping
-{%     elif ip_igmp_snooping.vlans is arista.avd.defined %}
+{% elif ip_igmp_snooping.vlans is arista.avd.defined %}
 !
-{%         for vlan in ip_igmp_snooping.vlans | arista.avd.natural_sort %}
-{%             if ip_igmp_snooping.vlans[vlan].enabled is arista.avd.defined(false) %}
+{%     for vlan in ip_igmp_snooping.vlans | arista.avd.natural_sort %}
+{%         if vlan.enabled is arista.avd.defined(false) %}
```

Comment:
this will not work for dict of dicts. only for list of dicts. Please change back to `if ip_igmp_snooping.vlans[vlan].enabled is arista.avd.defined(false)`